.github/workflows/package-creation-ecr.yaml

```diff
@@ -56,9 +58,11 @@ jobs:
         uses: aws-actions/amazon-ecr-login@v1
 
       - name: Setup image tag
+        id: setup-image-tag
+        shell: bash
         run: |
           echo "IMAGE_TAG=${GITHUB_SHA::8}" >> "${GITHUB_ENV}"
-        shell: bash
+          echo "::set-output name=image_tag::${{ env.IMAGE_TAG }}"
```

Comment:
@wayne-root  Can you please confirm if the run command fails in case the first command fails. Otherwise separate it out with another run command.

Reply:
Have split it into its own job